docs/tutorials/send-receive/ingress.md

```diff
@@ -0,0 +1,199 @@
+```
+### Deploy the operator
+
+#### create the namespace
+
+```console
+$ kubectl create namespace send-receive-project
+$ kubectl config set-context --current --namespace=send-receive-project
+```
+
+Wait for the Operator to start (status: `running`).
+```console
+$ kubectl get pod --namespace send-receive-project
+```
+
+### Deploy the ActiveMQ Artemis Broker
+
+For this tutorial we need to:
+
+* have a broker that is able to listen to any network interface. For that we
+  setup an `acceptor` that will be listening on every interfaces on port
+  `62626`.
+* have the ssl protocol configured for the `acceptor`
+* have queues to exchange messages on. These are configured by the broker
+  properties. Two queues are setup, one called `APP.JOBS` that is of type
+  `ANYCAST` and one called `APP.COMMANDS` that is of type `MULTICAST`.
+
+#### Create the certs
+
+We'll take some inspiration from the [ssl broker
+setup](https://github.com/artemiscloud/send-receive-project/blob/main/docs/tutorials/ssl_broker_setup.md)
+to configure the certificates.
+
+> [!NOTE]
+> In this tutorial:
+> * The password used for the certificates is `000000`.
+> * The secret name is `send-receive-sslacceptor-secret` composed from the broker
+>   name `send-receive` and the acceptor name `sselacceptor`
+
+```console
+$ keytool -genkeypair -alias artemis -keyalg RSA -keysize 2048 -storetype PKCS12 -keystore broker.ks -validity 3000
+$ keytool -export -alias artemis -file broker.cert -keystore broker.ks
+$ keytool -import -v -trustcacerts -alias artemis -file broker.cert -keystore client.ts
+$ kubectl create secret generic send-receive-sslacceptor-secret --from-file=broker.ks --from-file=client.ts --from-literal=keyStorePassword='000000' --from-literal=trustStorePassword='000000' -n send-receive-project
+```
+
+#### Start the broker
+
+```console
+$ kubectl apply -f - <<EOF
+apiVersion: broker.amq.io/v1beta1
+kind: ActiveMQArtemis
+metadata:
+  name: send-receive
+  namespace: send-receive-project
+spec:
+  ingressDomain: localresolvconf.com
+  acceptors:
+    - name: sslacceptor
+      port: 62626
+      expose: true
+      sslEnabled: true
+      sslSecret: send-receive-sslacceptor-secret
+  brokerProperties:
+    - addressConfigurations."APP.JOBS".routingTypes=ANYCAST
+    - addressConfigurations."APP.JOBS".queueConfigs."APP.JOBS".routingType=ANYCAST
+    - addressConfigurations."APP.COMMANDS".routingTypes=MULTICAST
+EOF
+```
+
+Wait for the Broker to start (status: `running`).
+
+```console
+$ kubectl get pod --namespace send-receive-project
+NAME                                                   READY   STATUS    RESTARTS
+activemq-artemis-controller-manager-66dbbbb679-hfmsn   1/1     Running   5 (3d22h ago)
+send-receive-ss-0                                      1/1     Running   0
+```
```

Comment:
We should check the ready condition in the ActiveMQArtemis CR to wait for the broker to start. I created #885 but in the meantime we could use custom-columns, i.e.
```
kubectl get aa -o custom-columns="NAME:.metadata.name,READY:.status.conditions[?(@.type=='Ready')].status"
```

Reply:
Thanks, that's a very good idea. Since you've got already a PR for the feature, it might make more sense to rebase this one on top of it once it's merged and to directly use the new status.